thorlcr/activities/spill/thspillslave.cpp

```diff
@@ -138,7 +132,7 @@ class SpillSlaveActivity : public CSlaveActivity
         Owned<IFile> ifile = createIFile(fileName.str());
         offset_t sz = ifile->size();
         mb.append(getDataLinkCount()).append(compress?uncompressedBytesWritten:sz).append(sz);
-        unsigned crc = compress?~0:fileCRC.get();
+        unsigned crc = ~0;
         mb.append(crc);
```

Comment:
would be good to get rid of this unused serialization element at the same time?

Reply:
done - as part of fixing the thor problems.